class.jetpack-connection-banner.php

```diff
@@ -182,6 +185,7 @@ public static function enqueue_connect_button_scripts() {
 				'connectInPlaceUrl'     => Jetpack::admin_url( 'page=jetpack#/setup' ),
 				'dashboardUrl'          => Jetpack::admin_url( 'page=jetpack#/dashboard' ),
 				'plansPromptUrl'        => Jetpack::admin_url( 'page=jetpack#/plans-prompt' ),
+				'identity'             => $identity,
```

Comment:
Is there anything we wouldn't want to expose here from the `$identity`?

Reply:
Since at that point the user is not connected yet, I don't think we send a lot in there, short of the Anon ID we create for that user?

Reply:
YEs we only send the anon identity there. We don't have the .com identity yet.